### ISSUE TITLE
R Version conflict ?

### ISSUE THREAD

**Issue by roger-randria** [Hirlam/obsmon#25]
./install --local-install
Directories in the R library search path (in order of priority):
  > /disk1/user/rogerr/Newobsmon/utils/build/local_R_library/R-libs 
  > /tmp/Rtmpd5t3oE/temp_R_lib 
  > /usr/local/lib/R/site-library 
  > /usr/lib/R/site-library 
  > /usr/lib/R/library 

Gathering R-libraries used in the ".R" files in current dir and subdirs...Error in readRDS(file) : 
  cannot read workspace version 3 written by R 3.6.3; need R 3.5.0 or newer
Error in readRDS(file) : 
  cannot read workspace version 3 written by R 3.6.3; need R 3.5.0 or newer
Error in readRDS(file) : 
  cannot read workspace version 3 written by R 3.6.3; need R 3.5.0 or newer
Error in readRDS(file) : 
  cannot read workspace version 3 written by R 3.6.3; need R 3.5.0 or newer
Error in readRDS(file) : 
  cannot read workspace version 3 written by R 3.6.3; need R 3.5.0 or newer
Done.
  > The following 28 main R-libraries were found:
    * argparse 
    * bsplus 
    * Cairo 
    * DBI 
    * dbplyr 
    * dplyr 
    * flock 
    * futile.logger 
    * future 
    * future.apply 
    * ggplot2 
    * gridExtra 
    * leaflet 
    * plotly 
    * png 
    * promises 
    * pryr 
    * RColorBrewer 
    * RcppTOML 
    * reshape2 
    * RSQLite 
    * shiny 
    * shinycssloaders 
    * shinyjs 
    * stringi 
    * stringr 
    * tidyr 
    * V8 

Getting recursive dependency list: 
Error in readRDS(dest) : 
  cannot read workspace version 3 written by R 3.6.2; need R 3.5.0 or newer
Calls: available.packages -> readRDS
Execution halted

This is what I have got today. Please any advice? Thanks in advance.

**Comment by paulovcmedeiros** [comment]
Hi Roger!

What version of obsmon are you using? The code requires R >= 4.0.5.

**Comment by roger-randria** [comment]
Hi Paulo,
Indeed sorry to disturb you. There were a few reasons for the failure:
 -- I had a few versions of R installed.
 -- Then I tried to install a package which didn't have full components under my op system.
 -- I had to uninstall the full R package, and stop the access to the wrong software packages. 
 -- Reinstalling a higher version of R was easy after the above steps.

So, thanks for the support.
